java/client/src/main/java/glide/api/models/commands/bitmap/BitEncodingOption.java

```diff
@@ -0,0 +1,28 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.bitmap;
+
+import lombok.Getter;
+
+public class BitEncodingOption {
+    public interface BitEncoding {
+        String getEncoding();
+    }
+
+    public static class SignedEncoding implements BitEncoding {
+        @Getter
+        private final String encoding;
+
+        public SignedEncoding(long encodingLength) {
+            encoding = "i".concat(Long.toString(encodingLength));
```

Comment:
js style
```suggestion
            encoding = "i" + encodingLength;
```

Reply:
Why is this preferred, Isn't `concat` better than `+`?